assets/training/forecasting_demand/environments/automl-gpu/environment.yaml

```diff
@@ -0,0 +1,12 @@
+image:
+  name: azureml/curated/automl-gpu
```

Comment:
do you have MCR PR somewhere?

Reply:
Where do we have this repository?

Reply:
@vizhur we did not know anything about MCR PR. It was not mentioned in the component release doc. Do you have a complete doc/guideline to release components? Thanks.

Reply:
I have created the PR to the MCR repository: https://github.com/microsoft/mcr/pull/2694